addons/point_of_sale/models/pos_order.py

```diff
@@ -657,10 +657,17 @@ def create_picking(self):
     def _force_picking_done(self, picking):
         """Force picking in order to be set as done."""
         self.ensure_one()
-        picking.action_assign()
+        contains_tracked_products = any([(product_id.tracking != 'none') for product_id in self.lines.mapped('product_id')])
+
+        # do not reserve for tracked products, the user will have manually specified the serial/lot numbers
+        if contains_tracked_products:
+            picking.action_confirm()
```

Comment:
@nim-odoo done, indeed makes sense to be consistent. Thanks for taking a look at it!